app/src/main/java/com/github/se/polyfit/ui/components/IngredientsOverview/ListProducts.kt

```diff
@@ -0,0 +1,73 @@
+package com.github.se.polyfit.ui.components.IngredientsOverview
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.github.se.polyfit.ui.screen.IngredientsTMP
+import com.github.se.polyfit.ui.theme.getGradient
+
+@Composable
+fun ListProducts(ListProducts: List<IngredientsTMP>, modifier: Modifier) {
+  val gradient = getGradient(active = true)
+
+  LazyColumn(
+      modifier = modifier.testTag("ListProductColumn"),
+      horizontalAlignment = Alignment.CenterHorizontally) {
+        items(ListProducts) {
+          Card(
+              modifier = Modifier.fillMaxWidth(0.8f).padding(16.dp, 8.dp).testTag("ProductCard"),
+              colors =
+                  CardDefaults.cardColors(
+                      MaterialTheme.colorScheme.background, MaterialTheme.colorScheme.onSurface),
+              border = BorderStroke(2.dp, gradient),
+              elevation = CardDefaults.cardElevation(0.dp)) {
+                Text(
+                    text = it.name,
+                    modifier = Modifier.padding(32.dp, 8.dp).testTag("ProductName"),
+                    color = MaterialTheme.colorScheme.outline,
+                    fontSize = MaterialTheme.typography.headlineMedium.fontSize)
+
+                TextIngredient(
+                    value = it.servingSize,
+                    text = "Serving Size",
```

Comment:
try to use the String resource if possible to keep coherent with the rest of the project

Reply:
Yep, my bad